species-search-dump/dump_species_search_data.py

```diff
@@ -0,0 +1,91 @@
+import mysql.connector
+from pydantic import BaseModel, Field, model_serializer, FieldValidationInfo, ValidationError, field_validator
+from typing import List, Dict, Any
+import json
+from json import JSONEncoder
+
+
+class Species(BaseModel):
+    id: str = Field(alias="assembly.accession")
```

Comment:
you probably want genome uuid rather than assembly accession id here

Reply:
Sorry I should have said in the PR, that is the original script. I moved it from root to a dump directory so that I could setup a poetry config for it's dependencies. The script used to generate the metadata based dump is [species-search-dump/dump_species_search_data_from_metadata.py](https://github.com/Ensembl/ensembl-web-metadata-api/pull/14/files#diff-78a40164dcc6d80f75394b6898b5bdbb36ff2902463379a66f54318aaf679aa4)

Reply:
The new script used genome UUID for the id